includes/Classifai/Providers/Azure/TextToSpeech.php

```diff
@@ -436,10 +463,30 @@ private function get_default_settings() {
 			'voices'        => array(),
 			'voice'         => '',
 			'authenticated' => false,
+			'default'       => true,
 			'post-types'    => array(),
 		];
 	}
 
+	/**
+	 * Helper to get the settings and allow for settings default values.
+	 *
+	 * @param string|bool|mixed $index Optional. Name of the settings option index.
+	 *
+	 * @return string|array|mixed
+	 */
+	public function get_settings( $index = false ) {
```

Comment:
So this method is already defined in the parent `Provider` class. From comparing the two, the only difference is this one sets defaults properly, is that correct? If so, I'm inclined to make this change to the default method (which I think would involve adding a default `get_default_settings` method. That way we aren't overwriting that method here

Reply:
@dkotter I noticed that, but the default settings function has been declared as a `private` method on all the other classes, so it does not really call the method from the child class.
I will need to make changes to the other classes too. I don't think the default settings method needs to be `private` and can be declared as `public`, but wanted to check with you regarding that. If there are concerns then we can declare them as `protected` for this to work as expected.

Reply:
Yeah, I see no reason why this should be `private`. In looking at the history, this method was first added to the `ComputerVision` class here: https://github.com/10up/classifai/pull/264 and no reason I can see from that PR why this needs to be `private`. I believe this was then just copied/pasted to other classes. We should be fine to change this